spec/models/conversation_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: conversations
+#
+#  id         :bigint           not null, primary key
+#  name       :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+require 'rails_helper'
+
+RSpec.describe Conversation, type: :model do
```

Comment:
Spec missing case checking users from the conversation.

We need to make sure that `conversation.users` returns only users from the conversation